saleor/graphql/shipping/tests/test_shipping_zones_query.py

```diff
@@ -308,3 +308,32 @@ def test_query_shipping_zone_search_by_channels(
 
     assert data[0]["node"]["name"] == shipping_zone_usd.name
     assert data[0]["node"]["id"] == shipping_zone_usd_id
+
+
+def test_query_shipping_zone_search_by_channels_no_matter_of_input(
```

Comment:
Name seems inaccurate, the input **does** matter because in the input we filter results by channel. Maybe something like `test_query_shipping_zone_filter_by_channel`?